070-Integrations/100-oauth.mdx

```diff
@@ -39,19 +40,19 @@ The application must include the following query parameters:
 
 | Parameter       | Description                                                                                                                                                                              |
 | --------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| `client_id`     | The client ID of the application.                                                                                                                                                        |
-| `redirect_uri`  | The URL that Xata should redirect the user back to after the user signs in. This URL must be registered with the application.                                                            |
-| `response_type` | Must be set to `code`.                                                                                                                                                                   |
-| `scope`         | Scopes that the application is requesting. Can be repeated to request multiple scopes. Right now, the only supported scope is `admin:all` and can be ignored as it is the default scope. |
-| `state`         | A string that the application can use to maintain state between the request and the callback. Also used to prevent CSRF attacks.                                                         |
+| `client_id`     | This is the client ID of the application.                                                                                                                                                        |
+| `redirect_uri`  | This is the URL that Xata should redirect the user back to after the user signs in. This URL must be registered with the application.                                                            |
+| `response_type` | This must be set to `code`.                                                                                                                                                                   |
+| `scope`         | This returns the scopes that the application is requesting. Scopes define what actions or data the application can access or perform for the user. This can be repeated to request multiple scopes. Right now, the only supported scope is `admin:all` and it can be ignored as it is the default scope. |
+| `state`         | A string that the application can use to maintain state between the request and the callback. This is also used to prevent CSRF attacks.                                                         |
```

Comment:
These are parameters, not responses. Don't think the changes to `This is` provide anything and definetely we should not change it to `This returns the scopes`.